src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder.java

```diff
@@ -561,8 +572,8 @@ public ListBoxModel doFillClusterNameItems(
           return items;
         }
 
-        clusters.forEach(c -> items.add(c.getName()));
-        selectOption(items, clusterName);
+        clusters.forEach(c -> items.add(ClusterUtil.toNameAndZone(c)));
```

Comment:
Would it be useful to comment that if zone is not given as input that clusters from all zones will be filled? It wasn't clear to me this was the case until I reviewed ContainerClient. This might be solved if we do end up changing zone to an optional as suggested.

Reply:
@stephenashank Saw your comment on the optional. If we're going to just list all clusters by their names with zone soon in another PR, then this probably won't be necessary. Also, is zone then provided as empty string?

Reply:
Good catch, thanks, I agree that this would have been a good place to leave a comment on that implied behavior. See #74 where I remove zone selection from the UI entirely. That also means that we only need one method on the container client which lists all clusters for a given project. With [this line](https://github.com/jenkinsci/google-kubernetes-engine-plugin/pull/74/files#diff-f3e4313d792d35b10fcbc494a4136b39R483) now using `listAllClusters` it's clearer what the behavior here is.